api/src/main/java/org/openmrs/module/pharmacyapi/api/model/DrugGroup.java

```diff
@@ -1,12 +1,12 @@
 /**
```

Comment:
We need to check the latest requirements updates to see if this entity is still required